.github/workflows/version.yml

```diff
@@ -36,91 +37,83 @@ jobs:
     needs: ["lint"]
     runs-on: ubuntu-latest
     outputs:
-      files: ${{ steps.changed_files.outputs.files }}
+      files: ${{ steps.calc_file_list.outputs.changed_files }}
+      sign_matrix: ${{ steps.calc_file_list.outputs.result }}
     steps:
     - name: Checkout the repository
       uses: actions/checkout@v4
     
     - name: Get changed files for push
       if: github.event_name == 'push'
       id: changed_files_push
-      uses: jitterbit/get-changed-files@v1
+      uses: masesgroup/retrieve-changed-files@v3.0.0
```

Comment:
Have you used this action before?
I could not find in their docs if the list will be `["stable.json"]` or `["./stable.json"]`

For the sign job, that does not matter I guess, but for the upload job it will.

Reply:
Yes, I am using it in the OS repo.

From the runs on my fork it seems the former:

```
Run masesgroup/retrieve-changed-files@v3.0.0
Base commit: 187aa687e9241a6e25ece3599d5da1dd9db41b29
Head commit: 2a53621954b68608758cace72cb482b47f42b950
All: [".github/workflows/version.yml"]
Added: []
Modified: [".github/workflows/version.yml"]
Removed: []
Renamed: []
Added or modified: [".github/workflows/version.yml"]
```

Reply:
Perfect 👌